tests/test_goldens.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 Google LLC
```

Comment:
Nit: Should we bump date when we transfer files? (applies to this and others)

Reply:
I don't think so. The date stays the same from when it was authored and this is primarily a copy. See the blue callout box under https://opensource.google/documentation/reference/releasing/preparing#license-headers.